hawk/src/main/java/com/orhanobut/hawk/Hawk.java

```diff
@@ -345,8 +348,29 @@ public Chain(int capacity) {
     public boolean commit() {
       return HAWK.storage.put(items);
     }
+  }
+
+  public static Map<String, ?> getAllSharedPrefs() {
```

Comment:
This is good idea to fetch all items at once, but I think we should make it more generic, instead of only sharedprefs, Storage can have getAll(). That would be good to add tests for these as well

```java
public static Map<String, ?> getAllSharedPrefs() {
  if (HAWK.storage == null) {
    return  null;
  }
  return  HAWK.storage.getAll());
}
```